extensions/cornerstone-3d/src/utils/dicomRTAnnotationExport/RTStructureSet/RTSSReport.js

```diff
@@ -0,0 +1,259 @@
+import AnnotationToPointData from './measurements/AnnotationToPointData';
+import dcmjs from 'dcmjs';
+import { DicomMetadataStore } from '@ohif/core';
+
+const { DicomMetaDictionary } = dcmjs.data;
+
+export default class RTSSReport {
+  constructor() {}
+
+  /**
+   * Convert Cornerstone tool annotation data to RTSSReport report object
```

Comment:
> Convert Cornerstone tool annotation data

Only if its formatted in a specific way, I think this is actually "convert handles to RTSTRUCT". E.g. Freehand poly won't work.

As this is super limited right now and doesn't really make what people expect from a full RTSTRUCT (e.g. a set of multiple ROIContours containing many 2D Contours), this should all be moved to the TMTV extension?